### ISSUE TITLE
server returns bad ASIN on every supplied ASIN

### ISSUE THREAD

**Issue by WeisseTeetasse** [djdembeck/bragibooks#95]

**Comment by garlik82** [comment]
Where are you getting your ASIN? as far as I know only audible uk and audible us ASIN's are supported.

**Comment by WeisseTeetasse** [comment]
Audible.com, while I am in Europe, I used the English Version of the Audiobook and also tried Goodreads audible asin, are there any logfiles somewhere?

**Comment by djdembeck** [comment]
What are some of the asin's you are using?

**Comment by garlik82** [comment]
Is the book Herculean by Jeremy Robinson? what audible region did you get that ASIN? Try this one is this Herculean is the book B01BDIVSPS

**Comment by meadmachine** [comment]
Thanks for the reply. The ASIN you suggested failed twice, then ran my CPU up to max for about 20 minutes before a server error page came up. When I refreshed the process seems to have completed successfully.
I am indeed based in the UK so the ASINs are from the UK store.
Some other examples (as I try them) that have failed are:
Alex Berenson - John Wells Series
- The Midnight House B079XYHTFZ
- The Night Ranger B00DE6398C
- The Secret Soldier B0065LML00
- The Counterfeit Agent B00IA3WVGU
- The Shadow Patrol B0081HPUIG

**Comment by WeisseTeetasse** [comment]
I had the server error too yesterday.
The CPU running is when bragibooks is working.
I am using ASINs from audible.com now, and they all work. Not Goodreads, not amazon.com only audible.com

**Comment by djdembeck** [comment]
Correct, this is made clear in the README: https://github.com/djdembeck/bragibooks#-usage-
